### DISCUSSION TITLE
[#991928] Player's body doesn't rotate

### DISCUSSION THREAD

**Discussion by xKingDark** [Mojang/minecraft-editor#59] [Bugs]
**How to Reproduce**:
  **1.** Switch to Crosshair Mode
  **2.** Change perspective to Third Person and start moving your camera around
  
**Observed Result**:
https://user-images.githubusercontent.com/47607185/224097529-f3c1d4a0-e5c6-4c15-9595-b265cdee22c4.mp4

**Expected Result**:
Player's body rotates when the user moves the camera around

**Helpful Info**:
- Preview build version: `1.19.80.20`

**Comment by minecraftchloe** [comment]
Thank you! Updating title for internal tracking

**Comment by minecraftchloe** [comment]
Should be resolved in the current build or our next release! LMK if you see it come up again!
